arrows/qt/widgets/LoggerView.cxx

```diff
@@ -12,15 +12,16 @@ namespace kv = kwiver::vital;
 namespace kwiver {
 namespace arrows {
 namespace qt {
-// -----------------------------------------------------------------------------
+
```

Comment:
Possible misconfiguration? My uncrustify seems to be inserting a blank line (as expected/desired).

Note that uncrustify does not currently have the ability to add blank lines between namespaces and content but *not* between different levels of nested namespaces; it may be this is "expected" because you changed the configuration to avoid the addition of blank lines in the latter case? (While it would be nice to have that feature, I've usually elected to accept the blank lines between different levels of namespaces.)

The incorrect number of `-`s is expected; that's a missing feature. (Inserting a space between `//` and the `-`s was expected and intended, but uncrustify does not currently have the ability to then ensure the correct number of `-`s are present.)